ReactWindows/ReactNative.Shared/UIManager/BorderedCanvasManager.cs

```diff
@@ -14,14 +15,52 @@ namespace ReactNative.UIManager
     public abstract class BorderedCanvasManager<TCanvas> : BorderedViewParentManager<TCanvas>
         where TCanvas : Canvas
     {
+        protected override bool HasBorder(TCanvas view)
+        {
+            return GetBorder(view) != null;
+        }
+
+        private Border GetBorder(TCanvas view)
```

Comment:
>GetBorder [](start = 23, length = 9)

I can see this being problematic if the actual first child is a border for some custom native component.